docs/project-incubation-exit.md

```diff
@@ -0,0 +1,139 @@
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+# Project Incubation Exit Criteria
+
+## Introduction
+
+The Hyperledger Project has defined a [lifecycle](./project-lifecycle.md) for its
+projects but this definition does not specify in any detail what it
+takes for a project to be able to transition from the
+*Incubation* phase to the *Active* phase.
+
+This document is meant to fill this gap by defining a set of criteria to
+be considered before moving a project from *Incubation*
+to *Active*.
+
+It is important to note that *Active* in this case refers to
+the project itself rather than its product and it is therefore more
+about the maturity of process than the maturity of the product or
+General Availability (GA).
+
+For this reason this document defines two sets of exit criteria. The
+first one is made of requirements expected to be met by all projects
+before moving to *Active*. The second set lists examples of
+additional requirements typically defined at the onset of the project as
+goals to be met to exit incubation. There are expected to be documented
```

Comment:
`There are expected to be documented in a Proposal for a Hyperledger Improvement Project` is bad grammar. i am not sure what this was trying to say. Maybe `There` was supposed to be `These`.